packages/react-native/ReactCommon/jsc/JSCRuntime.cpp

```diff
@@ -398,6 +398,11 @@ JSCRuntime::JSCRuntime(JSGlobalContextRef ctx)
       stringCounter_(0)
 #endif
 {
+#ifndef NDEBUG
+  if (__builtin_available(macOS 13.3, iOS 16.4, tvOS 16.4, *)) {
```

Comment:
This diff is breaking CircleCI. I don't think `__builtin_available` is available for Android, and what's in ReactCommon is built by both the platform. cc. @cortinico, I think we have to revert this.

Reply:
Thanks for flagging. Yes this needs specific Android handling

Reply:
Oh! Didn't realize this was used by Android, thanks all for catching!